src/rest-server/src/models/v2/job/k8s.js

```diff
@@ -244,61 +142,74 @@ const convertTaskDetail = async (taskStatus, ports, logPathPrefix) => {
   };
 };
 
-const convertFrameworkDetail = async (framework, tags) => {
-  const attemptStatus = framework.status.attemptStatus;
-  // check fields which may be compressed
-  if (attemptStatus.taskRoleStatuses == null) {
-    attemptStatus.taskRoleStatuses = decompressField(
-      attemptStatus.taskRoleStatusesCompressed,
-    );
-  }
-
+const convertFrameworkDetail = async (
+  latestFramework,
+  currentFramework,
```

Comment:
>currentFramework [](start = 2, length = 16)

currentFramework may refer to the latest one sometimes, so use latestAttemptFramework and thisAttemptFramework?